### DISCUSSION TITLE
Is there any way to calculate the eigenvalues of a sparse matrix as quick as MATLAB as??

### DISCUSSION THREAD

**Discussion by zyhsjtu** [JuliaLang/julia#56725] [Linear Algebra]
I want to calculate the maximum eigenvalues of a sparse. Is there any way to calculate the eigenvalues of a sparse matrix?

**Comment by jishnub** [comment]
Perhaps using one of `Arpack.jl`, `ArnoldiMethods.jl` or `KrylovKit.jl`? Depending on the matrix structure (e.g. banded), there might be other options.
